src/core/formatters.ts

```diff
@@ -244,10 +244,14 @@ export function multiTime() {
  * List of directives can be found on: https://github.com/mbostock/d3/wiki/Time-Formatting#format
  *
  * @param {string} [specifier] The specifier for the formatter.
+ * @param {boolean} [useUTC] Displays time in UTC if true, local time if false.
  *
  * @returns {Formatter} A formatter for time/date values.
  */
-export function time(specifier: string): Formatter {
+export function time(specifier: string, useUTC?: boolean): Formatter {
```

Comment:
Please add the default value for this optional argument: `useUTC = DEFAULT_USE_UTC` and above, `const DEFAULT_USE_UTC = false;` (also update comment to indicate default it to not use utc)